infra/tempo/config/tempo.yaml

```diff
@@ -0,0 +1,62 @@
+metrics_generator_enabled: true
+search_enabled: true
+
+server:
+  http_listen_port: 3200
+
+distributor:
+  receivers: # this configuration will listen on all ports and protocols that tempo is capable of.
```

Comment:
Do we get any resource hit by enabling all the receivers? 🤔

Reply:
If we do, I think it'll be pretty minor although we could remove the ones we don't need if it becomes a problem